compat/src/test/java/io/questdb/compat/AbstractTest.java

```diff
@@ -120,13 +131,13 @@ public void assertSql(CairoEngine engine, CharSequence sql, CharSequence expecte
 
     @Before
     public void setUp() {
-        LOG.info().$("Starting test ").$(getClass().getSimpleName()).$('#').$(testName.getMethodName()).$();
+        LOG.info().$("Starting test ").utf8(getClass().getSimpleName()).$('#').utf8(testName.getMethodName()).$();
```

Comment:
`testName.getMethodName()` can be invalid utf-8, please use $invalid()

Reply:
`$invalid` is only for `Utf8Sequnce`, this one is already a String and thus can't be invalid UTF-8.